### PR TITLE
DWeb gateways have moved to a private beta

### PR DESCRIPTION
**Pull request by thibmeu**
Inform customer that dweb gateways have moved to a private beta, and are
not provisioning SSL certificates during this interval.

### CODE REVIEW

**Review by Lekensteyn — APPROVED** [review]

File: products/distributed-web/src/content/ethereum-gateway/connecting-your-website.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ pcx-content-type: tutorial
 
 # Connecting your Website
 
+    To improve our service, the Ethereum gateway has moved to a Private Beta.
```
Comment (lukevalenta) [inline products/distributed-web/src/content/ethereum-gateway/connecting-your-website.md]: Looks like this is rendering as a code block due to the indentation. Is that intentional? This also causes the link now to be displayed correctly.

File: products/distributed-web/src/content/ipfs-gateway/connecting-website.md [hunk 2]
```diff
@@ -133,6 +133,10 @@ recommended course.
 
 ## Make It All Secure
 
+    To improve our service, the IPFS gateway has moved to a Private Beta.
```
Comment (lukevalenta) [inline products/distributed-web/src/content/ipfs-gateway/connecting-website.md]: Same comment as above.

File: products/distributed-web/src/content/ethereum-gateway/connecting-your-website.md (hunk 1 above)
Comment (thibmeu) [reply products/distributed-web/src/content/ethereum-gateway/connecting-your-website.md]: updated to use a quote instead

File: products/distributed-web/src/content/ipfs-gateway/connecting-website.md (hunk 2 above)
Comment (thibmeu) [reply products/distributed-web/src/content/ipfs-gateway/connecting-website.md]: updated to use a quote instead

**Review by lukevalenta — APPROVED** [review]

**Pull request merged**